deisctl/units/deis-logger.service

```diff
@@ -15,3 +15,7 @@ RestartSec=5
 
 [Install]
 WantedBy=multi-user.target
+
+[X-Fleet]
+MachineMetadata="deisRole=general"
+MachineMetadata="deisRole=control"
```

Comment:
Does that have the same effect?  I had trouble finding where that is documented.

What I'm after here is "a unit can run on a 'general purpose node"_or_ a control node."

(Or alternatively, for certain other units, "a unit can run on a 'general purpose node"_or_ a worker node."